parallel-consumer-python/.gitignore

```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
good lord that's a long list

Reply:
I can remove most of this, this is just the default .gitignore GitHub gives for Python